README.md

```diff
@@ -1 +1,68 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-World
+
+> Set up a "Hello Microverse" project
+
+
+## Built With
+
+- Html, CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] Kindly update your README file. It's better if you fill out all of the relevant sections to your project because the README file is the documentation of your project and making it professional reflects how professional your project is ✨